DashKitUI/Views/DashSettingsView.swift

```diff
@@ -333,6 +375,33 @@ struct DashSettingsView<Model>: View where Model: DashSettingsViewModelProtocol
             )
         }
     }
+    
+    func reservoirText(for level: ReservoirLevel) -> String {
+        switch level {
+        case .aboveThreshold:
+            let quantity = HKQuantity(unit: .internationalUnit(), doubleValue: Pod.maximumReservoirReading)
+            let thresholdString = viewModel.reservoirVolumeFormatter.string(from: quantity, for: .internationalUnit(), includeUnit: false) ?? ""
+            let unitString = viewModel.reservoirVolumeFormatter.string(from: .internationalUnit(), forValue: Pod.maximumReservoirReading, avoidLineBreaking: true)
+            return String(format: LocalizedString("%1$@+ %2$@", comment: "Format string for reservoir level above max measurable threshold. (1: measurable reservoir threshold) (2: units)"),
+                          thresholdString, unitString)
+        case .valid(let value):
+            let quantity = HKQuantity(unit: .internationalUnit(), doubleValue: value)
+            return viewModel.reservoirVolumeFormatter.string(from: quantity, for: .internationalUnit()) ?? ""
+        }
+    }
+    
+    func reservoirColor(for level: ReservoirLevel) -> Color {
+        switch level {
+        case .aboveThreshold:
+            return insulinTintColor
+        case .valid(let value):
+            if value > 10 {
+                return insulinTintColor
+            } else {
+                return guidanceColors.warning
+            }
+        }
+    }
 }
```

Comment:
[nit] Should this logic live in the view model that just exposes a `String` or `Color`?

Reply:
I moved reservoirText, but reservoirColor depends on Environment, so I kept it here.